gitrevise/tui.py

```diff
@@ -212,6 +235,14 @@ def inner_main(args: Namespace, repo: Repository):
 
 
 def main(argv: Optional[List[str]] = None):
+    if argv is None:
+        argv = sys.argv[1:]
+        for i, arg in enumerate(argv):
+            if arg == "--":
+                break
+            if arg.endswith("-S") or arg == "--gpg-sign":
+                argv[i] += "="
```

Comment:
This makes me quite uncomfortable. I understand that git's behaviour for this flag is quite weird, and ideally I'd like to copy how it behaves, but this way of processing is error-prone.

e.g. this would add an `=` to a command line option like `git revise -m "apples-S"`, which would definitely be the wrong thing to do.

I think I'd be the most comfortable with making `-S/--gpg-sign` be a boolean flag for now, to have the behaviour people are expecting in the command line list, and adding the ability to explicitly specify a signing key later with a separate flag if needed.